app/views/protocols/_participant_list.html.haml

```diff
@@ -1,6 +1,6 @@
 %h3= t(:participant)[:list]
 #custom-toolbar
-  = link_to t(:participant)[:new_participant], new_participant_path(protocol_id: protocol.id), {class: 'btn btn-default btn-primary', id: 'new_participant_button', type: 'button', data: {toggle: 'modal', target: '#modal_place'}, remote: true}
+  %button.btn.btn-primary.new-participant{type: "button", protocol_id: protocol.id, aria: {label: "Create Participant"}}=t(:participant)[:new_participant]
```

Comment:
You should use data attributes to store data so: %button.btn.btn-primary.new-participant{type: "button", data: { protocol_id: protocol.id }, aria: {label: "Create Participant"}}